### ISSUE TITLE
Model the "service" on the roads

### ISSUE THREAD

**Issue by nichbarry** [usds/playbook#122]
Just like roads the service does not exist to benefit anybody but the users. Yes, the government can close the roads whenever they want. But there are enough roads now to take alternate routes (it's just a pain in the ass). Alternatively, people have legs and can walk. You get it?

There are toll workers and road builders who earn income from building the road. But once the road is built it is maintained by the government who built it and people use it freely. Yes, there are taxes, and sometimes the fastest roads have tolls, but when you use the roads, you do not think: "oh this is the governments road". It's everyone's; and if we want to throw a block party, we contact our local municipal government for approval.

In summary: the "service" is selfless
